src/python_testing/matter_testing_support.py

```diff
@@ -1492,19 +1492,35 @@ def parse_matter_test_args(argv: Optional[List[str]] = None) -> MatterTestConfig
 
     return convert_args_to_matter_config(parser.parse_known_args(argv)[0])
 
+def async_to_sync(awaitable):
+    "Call sync, the async function."
+    # Usage of new implementation of ChipDeviceController cause the necessary to not use asyncio.run,
+    # because the initialization of `asyncio.Lock` use the `asyncio.get_event_loop()` on init.
+    # Usage of `asyncio.run()` and `asyncio.get_event_loop()` are exclusive.
+    loop = asyncio.get_event_loop()
+    return loop.run_until_complete(awaitable)
+
+def generate_random_nodeid(excluded_nodeid: typing.Optional[typing.List] = None) -> int:
+    if excluded_nodeid:
+        while True:
+            node_id=random.randint(0, 2^64-1)
```

Comment:
Node ID 0 is not allowed (it means "unspecified").

There are some forbidden node IDs in the 0xFFFF_FFFx_xxxx_xxxx.

Also `2^64` is incorrect Python syntax and will lead to a very small number (66). Correct syntax is `2**64`.

```suggestion
            node_id=random.randint(1, 0xFFFF_FFEF_FFFF_FFFF)
```

The above suggestion is spec compliant.

Reply:
Thanks for comment.
your suggestion has been applied